opentelemetry-etw-metrics/examples/basic-metrics-with-various-instruments.rs

```diff
@@ -0,0 +1,226 @@
+//! run with `$ cargo run --example basic --all-features
```

Comment:
the file name does not match this.. maybe rename the file to basic.rs and advanced.rs

Reply:
Updated to `examples/basic.rs` and `examples/advanced.rs`